README.md

```diff
@@ -1,74 +1,231 @@
-![banner](./.meta/stronghold_beta.png)
+<!-- This README is based on the BEST-README-Template (https://github.com/othneildrew Best-README-Template) -->
+<div id="top"></div>
 
-[![status](https://img.shields.io/badge/Status-Beta-green.svg)](https://github.com/iotaledger/stronghold.rs)
-![Audit](https://github.com/iotaledger/stronghold.rs/workflows/Audit/badge.svg?branch=dev)
-![Test](https://github.com/iotaledger/stronghold.rs/workflows/Test/badge.svg)
-[![docs](https://img.shields.io/badge/Docs-Official-red.svg)](https://stronghold.docs.iota.org)
-[![coverage](https://coveralls.io/repos/github/iotaledger/stronghold.rs/badge.svg?branch=dev)](https://coveralls.io/github/iotaledger/stronghold.rs?branch=dev)
-[![dependency status](https://deps.rs/repo/github/iotaledger/stronghold.rs/status.svg)](https://deps.rs/repo/github/iotaledger/stronghold.rs)
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fiotaledger%2Fstronghold.rs.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fiotaledger%2Fstronghold.rs?ref=badge_shield)
+<!-- PROJECT SHIELDS -->
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![Apache 2.0 license][license-shield]][license-url]
+[![Discord][discord-shield]][discord-url]
+[![StackExchange][stackexchange-shield]][stackexchange-url]
+<!-- Add additional Badges. Some examples >
+![Format Badge](https://github.com/iotaledger/stronghold.rs/workflows/Format/badge.svg "Format Badge")
+![Audit Badge](https://github.com/iotaledger/stronghold.rs/workflows/Audit/badge.svg "Audit Badge")
+![Clippy Badge](https://github.com/iotaledger/stronghold.rs/workflows/Clippy/badge.svg "Clippy Badge")
+![BuildBadge](https://github.com/iotaledger/stronghold.rs/workflows/Build/badge.svg "Build Badge")
+![Test Badge](https://github.com/iotaledger/stronghold.rs/workflows/Test/badge.svg "Test Badge")
+![Coverage Badge](https://coveralls.io/repos/github/iotaledger/stronghold.rs/badge.svg "Coverage Badge")
 
-## Introduction
-[summary]: #summary
 
-**IOTA Stronghold** is a secure software implementation with the sole purpose of isolating digital secrets from exposure to hackers and accidental leaks. It uses encrypted snapshots that can be easily backed up and securely shared between devices. Written in stable rust, it has strong guarantees of memory safety and process integrity. 
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+    <a href="https://github.com/iotaledger/stronghold.rs">
+        <img src="https://raw.githubusercontent.com/iotaledger/stronghold.rs/dev/.meta/stronghold_beta.png" alt="Banner">
+    </a>
+    <h3 align="center">Stronghold</h3>
+    <p align="center">
+        A Rust library to build a secure software storage for sensitive data.
+        <br />
+        <a href="https://wiki.iota.org"><strong>Explore the docs »</strong></a>
```

Comment:
```suggestion
        <a href="https://wiki.iota.org/stronghold.rs/welcome"><strong>Explore the docs »</strong></a>
```
This could lead to the Stronghold docs directly.